kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/UserCredentialsForm.vue

```diff
@@ -13,7 +13,12 @@
          inside a slot default
     -->
     <slot name="form">
+      <!-- Hiding the fullname and username textboxes, but their values are filled in and presumed
+           valid if we're given the user that we're taking credentials for (ie, just entering
```

Comment:
> presumed valid

if there is an error, what happens? do we get the validation errors and the form with fill-in textboxes again?

Reply:
Basically, IMO, this can use some refactoring because it basically uses one form in two use cases:
- Get a password for an imported user (ie, we imported a facility and they want to use an existing super admin account on that faciliy, so we get the password from them)
- Creating a whole new superuser

The convenient thing is that in the end the `SettingUpKolibri` component gets the same value from the same place -- but it might just be more clear if we separate those two needs into separate components and data storage (and/or state machine context?)

*But to answer the actual question...*

So the assumption here is that `username` and `fullname` are valid primarily because they are values that we got from another Kolibri in this particular case where we hide the form elements.